.github/CODEOWNERS

```diff
@@ -1,5 +1,2 @@
 # Every PR gets a review from an internal Expensify engineer
 * @Expensify/pullerbear
-
```

Comment:
I'm guessing all the various markdown edits are an artifact of something upstream?